sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1561,9 +1563,9 @@ object CodeGenerator extends Logging {
   private def logGeneratedCode(code: CodeAndComment): Unit = {
     val maxLines = SQLConf.get.loggingMaxLinesForCodegen
     if (Utils.isTesting) {
-      logError(s"\n${CodeFormatter.format(code, maxLines)}")
+      logError(log"\n${MDC(FORMATTED_CODE, CodeFormatter.format(code, maxLines))}")
```

Comment:
There are discussions around this one in https://github.com/apache/spark/pull/45851

Reply:
@dtenedor let's skip this one. The rest of the PR LGTM

Reply:
Sounds good, this is done.